packages/e2e-cypress/README.md

```diff
@@ -181,6 +181,20 @@ The good news is that we don't actually need to, since official documentation fo
 - [Vuex](https://docs.cypress.io/guides/component-testing/custom-mount-vue#Vuex)
 - [Pinia](https://pinia.vuejs.org/cookbook/testing.html#unit-testing-components)
 
+#### Using boot files
+
+When testing components, your Quasar boot files aren't loaded. If you need some features defined in your boot files, you can ss them in your support file.
+
+For instance, you can add some `globalProperties` to the `VueTestUtils` `config.global.mocks` object:
+
+```ts
+import { VueTestUtils } from 'cypress/vue';
+
+VueTestUtils.config.global.mocks.$sayHello = () => {
```

Comment:
It would be better to make a real-world scenario
Which was your original use case? If it's common enough, we can use that one
Otherwise we can use axios, etc

Reply:
Sure, I used the custom axios API example